triagebot.toml

```diff
@@ -424,7 +424,15 @@ message_on_reopen = "Issue #{number} has been reopened. Pinging @*T-types*."
 
 [notify-zulip."A-edition-2021"]
```

Comment:
Should 2021 be removed or at least discriminated?

Reply:
I appreciate continuing to receive notifications, since they are still being filed.

Reply:
Oh, but I see what you're saying about noting the difference between them. Yea, I can add that, thanks!